code-challenges.test.js

```diff
@@ -34,9 +63,31 @@ const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+describe("modulo", () => {
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", () => {
+    expect(modulo(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(modulo(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
 
 // b) Create the function that makes the test pass.
 
+// Pseudo Code:
+// Create a function that takes an array as a parameter. (modulo)
+// Create a variable of array with a method applied, that returns the value if it is equivalent to a number. [filter] (numbersArr)
+// Create a variable of the filtered array with a method applied, that returns an array of the value modulo 3. [map] (remaindersArr)
+// Have the function return the mapped array variable.
+// Run the function with the provided variables.
+
+function modulo(arr) {
+  let numbersArr = arr.filter(value => typeof value == 'number')
+  let remaindersArr = numbers.map(value => value % 3)
+  return remaindersArr
+}
+
+console.log(modulo(hodgepodge1))
+
+console.log(modulo(hodgepodge2))
```

Comment:
Great job, Sam!  Just one small thing here - line 84, you used numbers instead of numbersArr.  This made the test fail.  Just a minor mistake. Your code is clean and efficient.  Beautiful.  Great pseudocode again.